packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -330,6 +332,26 @@ export default class QuanticCategoryFacet extends LightningElement {
     this.dispatchEvent(renderFacetEvent);
   }
 
+  validateDependsOnProperty() {
+    if (this.dependsOn) {
+      getBueno(this).then(() => {
+        const {parentFacetId, expectedValue} = this.dependsOn;
+        if (!parentFacetId || !Bueno.isString(parentFacetId)) {
```

Comment:
Can we simply do `!Bueno.isString(parentFacetId)` is Bueno checking first if the value is not null?

Reply:
good point, if the value is null or undefined  it's not a string 👍 updated